dpgen/generator/lib/make_calypso.py

```diff
@@ -95,18 +96,19 @@ def make_calypso_input(nameofatoms,numberofatoms,
         ret+= "###################End Parameters for VSC ##########################\n"
     return ret
 
-def _make_model_devi_buffet(jdata,calypso_run_opt_path):
+def _make_model_devi_buffet(jdata,caly_run_opt_path):
```

Comment:
I would prefer the original variable name, which is self-explanatory to the readers. It is not easy to relate `caly` to `calypso`...